xml/deploy_cephadm.xml

```diff
@@ -2188,17 +2184,13 @@ spec:
       </para>
      </listitem>
      <listitem>
-      <para>
-       <replaceable>EXAMPLE_NAMESPACE</replaceable> (optional) with the desired
-       NFS namespace (for example, <literal>nfs</literal>). For more
-       information on NFS namespaces, see
-       <link xlink:href="https://docs.ceph.com/docs/master/radosgw/nfs/#namespace-conventions
-"/>
-<!-- 2020-06-10 tbazant: possibly import that upstream section into
-       Admin Guide? -->
-      </para>
+       <para>
+         <replaceable>EXAMPLE_NAMESPACE</replaceable> (optional) with the desired
+         NFS namespace (for example, <literal>nfs</literal>). For more
+         information on NFS namespaces, see <xref linkend="ceph-nfsganesha-namespace"/>.
```

Comment:
Just like the previous comment from @mgfritch, the namespace here is a rados namespace, not a RGW namespace. So the reference is not appropriate.